### ISSUE TITLE
July 2020 PythonPune Newsletter Collection Box

### ISSUE THREAD

**Issue by digitronik** [pythonpune/meetup-talks#109]
Let's use this issue to collect python related news, new package updates and events happening around Pune. We can compile and publish it during **July PythonPune** meetup. 

## How to add news:
Comment on this issue with news link and news topic and also include the source from we got this. 

## How to add new package release:
Paste the link of new package release

## How to include events here:
If a new event is happening in Pune and wants to share in the newsletter, feel free to drop the event link with date and venue.

**Comment by digitronik** [comment]
Pytest major release (pytest-6.0) coming soon. Pre-release rc1 available for testing.
https://pypi.org/project/pytest/6.0.0rc1/
https://docs.pytest.org/en/latest/changelog.html#pytest-6-0-0rc1-2020-07-08

**Comment by digitronik** [comment]
Now, Github Action supporting unstable python versions (Python-3.9 beta).
https://github.com/actions/python-versions/pull/38
https://github.com/actions/python-versions/releases/tag/3.9.0-beta.4-20200715.2

**Comment by digitronik** [comment]
Python 3.8.4 is now available
https://www.python.org/downloads/release/python-384/

**Comment by digitronik** [comment]
Pip team midyear report
http://pyfound.blogspot.com/2020/07/pip-team-midyear-report.html

**Comment by amolkahat** [comment]
Manim: Mathematical Animation Engine
Github: https://github.com/3b1b/manim
Tutorial: https://talkingphysics.wordpress.com/2019/01/08/getting-started-animating-with-manim-and-python-3-7/

**Comment by raukadah** [comment]
* Run Your Flask Regularly Scheduled Jobs with Cron https://blog.miguelgrinberg.com/post/run-your-flask-regularly-scheduled-jobs-with-cron
tldr:
* A little introduction about cron tool and scheduling job
* How to implement the job logic by seperating the schedular and the actual application
* How to use crontab?
* How to have proper logging for the scheduled job?

**Comment by raukadah** [comment]
Introducing pyage-rust, a Python module for age encryption https://kushaldas.in/posts/introducing-pyage-rust-a-python-module-for-age-encryption.html
tldr:
* Python module for age ( simple, modern and secure file encryption tool,), built on top of rust crate (rage)
* Install it using python3 -m pip install pyage-rust==0.1.0
* check the biog on how to encrypt or decrypt

**Comment by raukadah** [comment]
Debugging ImportError and ModuleNotFoundErrors in your Docker image https://pythonspeed.com/articles/importerror-docker/
 tldr:
 * Using the correct python interpreters
 * virtualenv is correctly activated
 * Either install all code via pip
 * OR keep all code where python -m is getting executed

**Comment by raukadah** [comment]
https://realpython.com/python-import/ Python import: Advanced Techniques and Tips
 tldr:
 * Difference between absolute and relative imports
 * resource vs Dynamic imports
 * How python import system works
 * Some tips & tricks regarding handling imports across multiple versions

**Comment by raukadah** [comment]
Episode 18: Ten Years of Flask: Conversation With Creator Armin Ronacher https://realpython.com/podcasts/rpp/18/

**Comment by raukadah** [comment]
Project Releases:
 Announcing Poetry 1.0.10 https://python-poetry.org/blog/announcing-poetry-1-0-10.html
 tldr:
 * init and new commands will now provide hints on invalid given licenses
 
 urllib3-1.25.10 is released https://github.com/urllib3/urllib3/blob/1.25-series/CHANGES.rst#12510-2020-07-22
 
 3.9.0b5 the last beta before 3.9.0 is available
 https://discuss.python.org/t/python-3-8-5-released-as-a-security-hotfix-3-9-0b5-the-last-beta-before-3-9-0-also-available/4743

**Comment by raukadah** [comment]
Toolz (A set of utility functions for iterators, functions, and dictionaries.) https://github.com/pytoolz/toolz

**Comment by bhavin192** [comment]
Closed by #116
